A hack to get around differences between Sun javah and gcj javah - in Sun
javah, symbols in a nested class use _ as a separator instead of _00024.
Patch is originally from Nobuhiro Iwamatsu, bug #561516.

--- a/build/generator/templates/makefile.ezt
+++ b/build/generator/templates/makefile.ezt
@@ -99,7 +99,8 @@
 [if-any target.headers][target.varname]_CLASS_FILENAMES =[for target.header_class_filenames] [target.header_class_filenames][end]
 [target.varname]_CLASSES =[for target.header_classes] [target.header_classes][end]
 $([target.varname]_HEADERS): $([target.varname]_CLASS_FILENAMES)
-	[target.link_cmd] -d [target.output_dir] -classpath [target.classes]:$([target.varname]_CLASSPATH) $([target.varname]_CLASSES)
+	[target.link_cmd] -d [target.output_dir] -classpath [target.classes]:$([target.varname]_CLASSPATH) -all [target.classes]
+	cd [target.output_dir]; sed -i s/_00024/_/ *.h
 [end][if-any target.sources][target.varname]_SRC =[for target.sources] [target.sources][end]
 $([target.varname]_OBJECTS): $([target.varname]_SRC)
 	[target.link_cmd] -d [target.output_dir] -classpath [target.classes]:$([target.varname]_CLASSPATH) $([target.varname]_SRC)
